libnacl/sealed.py

```diff
@@ -0,0 +1,33 @@
+import libnacl
+import libnacl.public
+import libnacl.dual
+
+class SealedBox(object):
```

Comment:
Sealed box doesn't have special keys, it support the same key objects as normal box (as it use box underneath). I see that libnacl.public.Box also doesn't have those key management functions inherited so I don't think SealedBox would be any different.